Content.Server/_White/Abilities/Psionics/Abilities/ShadowSprintSystem.cs

```diff
@@ -0,0 +1,60 @@
+using Content.Shared.Abilities.Psionics;
+using Content.Shared.Psionics;
+using Content.Shared.Shadowkin;
+using Robust.Server.GameObjects;
+using Robust.Shared.Timing;
+using Content.Shared.Movement.Components;
+using Content.Shared.Movement.Systems;
+using Content.Shared._White.Psionics.Events;
+
+namespace Content.Server._White.Abilities.Psionics
```

Comment:
```suggestion
namespace Content.Server._White.Abilities.Psionics.Abilities;
```
Неправильный неймспейс